FerramAerospaceResearch/LEGACYferram4/FARWingAerodynamicModel.cs

```diff
@@ -1476,7 +1476,7 @@ public override void OnLoad(ConfigNode node)
                 int.TryParse(node.GetValue("nonSideAttach"), out nonSideAttach);
             if (node.HasValue("MidChordSweep"))
                 double.TryParse(node.GetValue("MidChordSweep"), out MidChordSweep);
-            if (node.HasValue("MassOverride") && float.TryParse(node.GetValue("MassOverride"), out float mass))
+            if (node.HasValue("massOverride") && float.TryParse(node.GetValue("massOverride"), out float mass))
```

Comment:
Keep PascalCase in line with other values

Reply:
That would mean that the KSPField should be renamed to PascalCase too? (As I understand it, they have to be the same)

Reply:
If that's the case then it's fine